### ISSUE TITLE
[Add-on] Private relay addon's injection of "relay.css" breaks the layout at linkedin.com

### ISSUE THREAD

**Issue by dholbert** [mozilla/fx-private-relay#551]
If the Private Relay add-on is installed, then the layout at https://www.linkedin.com/psettings/ is broken.

(Note: you need to have a LinkedIn account and be signed in in order to see this page.)

Specifically:
 - the header ($YOUR_NAME, you're the boss of your account) is clipped at the bottom, and the avatar that shows up there instead is clipped & superimposed with the section listings below it.
 - there's a useless horizontal scrollbar (even in my maximized browser window on a plenty-large laptop display)

Screenshot with broken UI:
![image](https://user-images.githubusercontent.com/426803/87608140-faf49500-c6b3-11ea-87a1-a693e436cac5.png)

Screenshot of expected rendering (if I remove the add-on):
![image](https://user-images.githubusercontent.com/426803/87608229-2c6d6080-c6b4-11ea-9f6f-0898cdb6fcec.png)

**Comment by dholbert** [comment]
The header-squashing part (at least) is due to this rule from `relay.css` which the add-on injects into the page somehow (as I can see in devtools):
```css
.settings-header, logo-wrapper {
    display: flex;
    flex-direction: row;
    justify-content: flex-start;
    align-items: center;
    padding: 0 24px 0 24px;
    width: 100%;
    border-bottom: 1px solid var(--grey20);
    min-height: 64px;
    height: 64px;
    max-height: 64px;
}
```
...from moz-extension://746afece-f2a2-4cc3-929b-be88a2d5259b/css/relay.css

LinkedIn's header happens to use class name `.settings-header`, so this rule matches and wreaks havoc. (The 'height' declarations in particular are problematic and cause it to be short, but I'm sure the rest is troublesome too.)

**Comment by dholbert** [comment]
I'm using add-on version 1.4.1 , which is the latest available at https://addons.mozilla.org/en-US/firefox/addon/private-relay/

Superficially this seems like the same approximate issue as https://github.com/mozilla/fx-private-relay/issues/472 , but that was supposedly fixed last month... @groovecoder do you know what's going on?

**Comment by dholbert** [comment]
Reduced testcase:
https://jsfiddle.net/dholbert/kay5bw6t/

Source:
```css
.settings-header {
  background: red;
  width: 600px;
  height: 0;
}
```
```html
No red should be visible.
<div class="settings-header"></div>
```

**Comment by groovecoder** [comment]
This should be fixed in 1.4.2 which is already submitted to AMO. We moved the `settings-header` class definition into `popup.css` which will only load in `popup.html`

**Comment by groovecoder** [comment]
Fixed in 1.4.3 which is now released on AMO.

**Comment by dholbert** [comment]
Confirmed, this looks good now with v1.4.3.

Thanks!
